### DISCUSSION TITLE
Why is 172.17.0.1 the default value of the bridge network?

### DISCUSSION THREAD

**Discussion by adrian-sa** [moby/moby#46033] [Q&A]
Hi !

I understand that 172.17.0.1 is the default value for the bridge network in docker.

However i couldn't find the "why" in the documentation.
Also, where is it defined in the moby repo, i tried finding where that value is defined

Thanks !

**Comment by thaJeztah** [comment]
The daemon uses IP ranges that are reserved for Private networks (see [RFC 1918, Section 3](https://datatracker.ietf.org/doc/html/rfc1918#section-3). Those ranges are defined here; https://github.com/moby/moby/blob/454b6a7cf5187d1153159e5fe07f4b25c7a95e7f/libnetwork/ipamutils/utils.go#L11-L28
